guides/source/active_job_basics.md

```diff
@@ -280,6 +280,18 @@ UserMailer.welcome(@user).deliver_later
 ```
 
 
+Internationalization
+--------------------
+
+Each job will use the same I18n.locale that was set while it the job was created.
```

Comment:
I'd rephrase as `Each job uses the `I18n.locale` set when the job was created. Useful if you send emails asynchronously:`